Cargo.toml

```diff
@@ -13,8 +13,15 @@ edition = "2018"
 [features]
 default = ["bus", "journal"]
 
+full = [
```

Comment:
I think it's a better pattern long term to require consumers to opt in to the latest version they want via feature flags.

Adding this `full` flag will require a debate every time a new systemd comes out around whether we can bump the `full` default to include it or not.

A good contrasting example is e.g. https://github.com/ostreedev/ostree-rs/blob/45ee704578432e757768311ab6d0072571333911/Cargo.toml#L56
Which is the pattern for the glib-rs ecosystem.

Reply:
Tokio provides the feature flag `full` (https://github.com/tokio-rs/tokio/blob/tokio-1.6.1/tokio/Cargo.toml#L29-L43) so I thought it is a good idea to add it here as well. But version flags are probably a different thing so I will remove it.